src/Microsoft.Identity.Client/Internal/Requests/InteractiveRequest.cs

```diff
@@ -270,5 +276,29 @@ private void VerifyAuthorizationResult()
                 throw new MsalServiceException(_authorizationResult.Error, _authorizationResult.ErrorDescription, null);
             }
         }
+
+        private async Task CheckForBrokerAndAcquireAuthorizationAsync(CancellationToken cancellationToken)
+        {
+            Broker = brokerFactory.CreateBrokerFacade(ServiceBundle.DefaultLogger);
+
+            if (Broker.CanInvokeBroker(_interactiveParameters.UiParent, ServiceBundle))
+            {
+                _msalTokenResponse = await Broker.AcquireTokenUsingBrokerAsync(
+                    CreateBrokerPayload(),
+                    ServiceBundle).ConfigureAwait(false);
+            }
+            else
+            {
+                await AcquireAuthorizationAsync().ConfigureAwait(false);
+                VerifyAuthorizationResult();
+                _msalTokenResponse = await SendTokenRequestAsync(GetBodyParameters(), cancellationToken).ConfigureAwait(false);
+            }
+        }
+
+        private Dictionary<string, string> CreateBrokerPayload()
```

Comment:
Is this method needed? if a variable is beneficial for debugging consider creating a var where it's used. At this time nothing else happens in this method.